### PR TITLE
Add compose example for DocSum amd rocm deployment

### PR DESCRIPTION
**Pull request by artem-astafev**
## Type of change

- Added compose file for AMD ROCm deployment.
- Added Readme file for AMD ROCm deployment example.
- Added tests for AMD ROCm deployment

## Tests

Go to GenAIExamples/DocSum/tests and run test_compose_on_rocm.sh test(./test_compose_on_rocm.sh test) script

### CODE REVIEW

File: DocSum/tests/test_compose_on_rocm.sh [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+export BACKEND_SERVICE_ENDPOINT="http://${ip_address}:8888/v1/docsum"
+export DOCSUM_CARD_ID="card1"
+export DOCSUM_RENDER_ID="renderD136"
+
+function build_docker_images() {
+    cd "$WORKPATH"/docker_image_build
+    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+
+    echo "Build all the images with --no-cache, check docker_image_build.log for details..."
+    docker compose -f build.yaml build --no-cache > "${LOG_PATH}"/docker_image_build.log
```
Comment (chensuyue) [inline DocSum/tests/test_compose_on_rocm.sh:39]: No need to build all the images, just build the images required by this test, https://github.com/opea-project/GenAIExamples/blob/82f418bf3439672ce570d05cbe2a1ca257b3d6b8/DocSum/tests/test_compose_on_gaudi.sh#L22

File: DocSum/tests/test_compose_on_rocm.sh [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+function stop_docker() {
+    cd "$WORKPATH"/docker_compose/amd/gpu/rocm
+    docker compose stop && docker compose rm -f
+}
+
+function main() {
+
+    stop_docker
+
+#    if [[ "$IMAGE_REPO" == "opea" ]]; then build_docker_images; fi
```
Comment (chensuyue) [inline DocSum/tests/test_compose_on_rocm.sh:178]: We should open the image build, so the images will be build based on the PR. Sometimes a PR changed the dockerfile and scripts, CI need to verify it on all the supported hardware.

File: DocSum/docker_compose/amd/gpu/rocm/compose.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    cap_add:
+      - SYS_PTRACE
+    group_add:
+      - video
+    security_opt:
+      - seccomp:unconfined
+    ipc: host
+    command: --model-id ${DOCSUM_LLM_MODEL_ID}
+  docsum-llm-server:
+    image: ${REGISTRY:-opea}/llm-docsum-tgi:latest
```
Comment (chensuyue) [inline DocSum/docker_compose/amd/gpu/rocm/compose.yaml:32]: ${REGISTRY:-opea}/llm-docsum-tgi:${TAG:-latest}, do not freeze the tag.

File: DocSum/docker_compose/amd/gpu/rocm/README.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
+## 🚀 Start Microservices and MegaService
```
Comment (chensuyue) [inline DocSum/docker_compose/amd/gpu/rocm/README.md:1]: ## -> #  will resolve this https://github.com/opea-project/GenAIExamples/actions/runs/11815151412/job/32932891261?pr=1125

Comment (artem-astafev) [reply DocSum/docker_compose/amd/gpu/rocm/README.md]: Ok, thank you for the tip.

File: DocSum/tests/test_compose_on_rocm.sh (hunk 1 above)
Comment (artem-astafev) [reply DocSum/tests/test_compose_on_rocm.sh]: done

File: DocSum/tests/test_compose_on_rocm.sh (hunk 2 above)
Comment (artem-astafev) [reply DocSum/tests/test_compose_on_rocm.sh]: revert docker build, done

File: DocSum/docker_compose/amd/gpu/rocm/compose.yaml (hunk 3 above)
Comment (artem-astafev) [reply DocSum/docker_compose/amd/gpu/rocm/compose.yaml]: revert tags, done

**Comment by artem-astafev** [comment]
Seems like test is failing for backend service because of https://github.com/opea-project/GenAIExamples/issues/1129

**Review by lkk12014402 — APPROVED** [review]

**Review by chensuyue — APPROVED** [review]

**Pull request merged**